scripts/deb-builder-helpers.sh

```diff
@@ -33,19 +33,19 @@ TEST_EXECUTIVE_DEPS=", mina-logproc, python3, nodejs, yarn, google-cloud-sdk, ku
 
 case "${MINA_DEB_CODENAME}" in
   bookworm|jammy)
-    DAEMON_DEPS=", libffi8, libjemalloc2, libpq-dev, libprocps8, mina-logproc"
+    DAEMON_DEPS=", libffi8, libjemalloc2, libpq-dev, libprocps8, mina-logproc, mina-create-legacy-genesis"
```

Comment:
This dependency is also used in zkapp_test_transaction debian where it is not used. Can we solve that by introducing another env for Zkapp_test_transaction or have an explicit dependency set for this debian.

Reply:
@dkijania such as this?